crates/calcite-schema/src/schema.rs

```diff
@@ -62,7 +62,11 @@ use crate::version5::ParsedConfiguration;
 // ANCHOR: get_schema
 #[tracing::instrument(skip(configuration, calcite_ref), level=Level::INFO)]
 pub fn get_schema(configuration: &ParsedConfiguration, calcite_ref: GlobalRef) -> Result<SchemaResponse> {
-    let data_models = get_models(&calcite_ref);
+    let data_models = match configuration.clone().metadata {
```

Comment:
This is not correct, if the `configuration.metadata` is nothing, then we should still not introspect the data source. The only place introspection of the data source can happen is through the cli `update` command.